include/psxsdk/libgpu.h

```diff
@@ -129,10 +129,17 @@ typedef struct {
     /* 0x6 */ short h; /* height */
 } RECT;                /* size = 0x8 */
 
+#ifdef VERSION_PC
+#define O_TAG                                                                  \
```

Comment:
I think some documentation explaining the O_TAG / OT_TYPE problem and solution would be good, my understanding is pretty low